### PR TITLE
DEV: Rely properly on `selenium-manager` for system specs

### PR DESCRIPTION
**Pull request by Flink**

### CODE REVIEW

**Comment by Flink** [comment]
@tgxworld We’ll have to wait for https://github.com/discourse/discourse_docker/pull/782 before merging this one. I’m also running some tests in https://github.com/discourse/discourse/pull/26244

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -568,22 +551,6 @@ class SpecTimeoutError < StandardError
...
     if !system_tests_initialized
-      # Use a file system lock to get `selenium-manager` to download the `chromedriver` binary that is required for
-      # system tests to support running system tests in multiple processes. If we don't download the `chromedriver` binary
-      # before running system tests in multiple processes, each process will end up calling the `selenium-manager` binary
-      # to download the `chromedriver` binary at the same time but the problem is that the binary is being downloaded to
-      # the same location and this can interfere with the running tests in another process.
-      #
-      # The long term fix here is to get `selenium-manager` to download the `chromedriver` binary to a unique path for each
-      # process but the `--cache-path` option for `selenium-manager` is currently not supported in `selenium-webdriver`.
-      File.open("#{Rails.root}/tmp/chrome_driver_flock", File::RDWR | File::CREAT, 0644) do |file|
```
Comment (Flink) [inline spec/rails_helper.rb:579]: We won’t need this anymore since the chrome driver will be installed by Debian at `/usr/bin/chromedriver`.

Comment (tgxworld) [reply spec/rails_helper.rb:579]: We will still need this for people running system tests locally with `turbo_rspec`. They can still run into problems if their chrome browser is updated and selenium-manager needs to install the chrome driver for that particular browser version.

Comment (Flink) [reply spec/rails_helper.rb:579]: Running one spec in isolation will take care of the problem in theory (the chrome driver will have been downloaded by selenium-manager then), but I guess it doesn’t hurt to keep it for now :slightly_smiling_face:

**Comment by Flink** [comment]
@tgxworld Actually we can merge this PR, it won’t break anything as it will be able to use either Chrome or Chromium.

**Review by tgxworld — APPROVED** [review]

**Comment by tgxworld** [comment]
Thank you for working on this @Flink 👍

**Pull request merged**